test/functional/cache/custom-cache-provider.ts

```diff
@@ -29,12 +29,12 @@ describe("custom cache provider", () => {
       expect(queryResultCache).to.have.property("queryResultCacheTable");
 
       const queryResultCacheTable = queryResultCache.queryResultCacheTable;
-      expect(queryResultCacheTable).to.contain("mock");
+      expect(queryResultCacheTable.toLowerCase()).to.contain("mock");
     })));
 
     it("should cache results properly", () => Promise.all(connections.map(async connection => {
 
-        // first prepare data - insert users
+        // first prepare data - insert user_cache
         const user1 = new User();
```

Comment:
Why are we changing the cache tests?

They don't seem related

Reply:
The trouble with Oracle is that it returns results in upper case. I know the escaping would mean it comes back in lowercase/camelcase but with the other issues experienced I need to find a way to take the raw results from Oracle and match what typeorm does. 

I am looking at how I can do this specifically for OracleDriver/QueryRunner only to not affect other things. You can safely ignore these ones as I have closed this down as it was getting unwieldy and not nice (sure you would agree) and working on a better solution